src/main/java/tornadofx/Builders.kt

```diff
@@ -109,6 +109,12 @@ fun Pane.progressbar(property: Property<Double>, op: (ProgressBar.() -> Unit)? =
     progressProperty().bind(property)
 }
 
+fun Pane.slider(min: Double? = null, max: Double? = null, value: Double? = null, op: (Slider.() -> Unit)? = null) = opcr(this, Slider().apply {
```

Comment:
I don't really see a problem with how you did it here, but if you wanted to invoke the other constructor I don't see the harm. 

You'll have to be the judge if it makes it too complicated or not. I suppose you could also just call the constructor with those 3 parameters always, and use 0 as a default values instead of null (I don't know if Kotlin will handle the null -> primitive conversion or throw an error). 

Either way, let me know when you're happy with it and I'll merge.